package.json

```diff
@@ -2,14 +2,21 @@
   "name": "single-spa-vue",
   "version": "1.10.1",
   "description": "a single-spa plugin for vue.js applications",
-  "main": "lib/single-spa-vue.js",
+  "main": "dist/umd/single-spa-vue.js",
+  "module": "dist/esm/single-spa-vue.js",
```

Comment:
This will make webpack and rollup use the ES version of single-spa-vue when bundling.